app/medInria/medWorkspaceArea.cpp

```diff
@@ -122,7 +102,120 @@ medWorkspaceArea::~medWorkspaceArea(void)
 
 QPixmap medWorkspaceArea::grabScreenshot()
 {
-    return QPixmap::grabWindow(this->currentWorkspace()->stackedViewContainers()->currentWidget()->winId());
+    medAbstractView* currentView = currentWorkspace()->stackedViewContainers()->getFirstSelectedContainerView();
+    if (currentView != nullptr)
+    {
+        return QPixmap::grabWindow(currentView->viewWidget()->winId());
+    }
+    return QPixmap();
+}
+
+void medWorkspaceArea::grabVideo()
+{
+    medAbstractView* view = this->currentWorkspace()->stackedViewContainers()->getFirstSelectedContainerView();
+    if (view)
+    {
+        medAbstractProcess* process = qobject_cast<medAbstractProcess*>(dtkAbstractProcessFactory::instance()->create("msc::ExportVideo"));
```

Comment:
```process``` never gets deleted (which will lead to big memory leaks since it stores the video data).